client-metrics-reporter/src/test/java/io/strimzi/kafka/metrics/prometheus/MetricsUtils.java

```diff
@@ -179,7 +194,24 @@ public static void startConnector(StrimziConnectCluster connect, String name, St
                     TimeUnit.MILLISECONDS.sleep(100L);
                 }
             }
+
+            while (true) {
```

Comment:
Also, here is a one-liner describing what we waiting...e.g., `// Retry task state check`

Reply:
Done